content/github/building-a-strong-community/blocking-a-user-from-your-personal-account.md

```diff
@@ -49,6 +49,9 @@ In repositories you own, blocked users also cannot:
 - Comment on issues, pull requests, or commits
 - Add or edit wiki pages
 
+(Keep in mind users you block will not receive email notifications of
+comments you make, even if [@mentioning](/articles/basic-writing-and-formatting-syntax/#mentioning-people-and-teams) them directly.)
+
```

Comment:
This is helpful clarification ✨ but it seems like it would be more beneficial in one of the lists that already exist, just to keep content all in one place. I suggest making it a bullet point in the "After you've blocked a user, they cannot:" list, close to the related information 😄 

> After you've blocked a user, they cannot:
> - Send you any notifications, including by @mentioning your username
> - Receive any notifications sent by you, including comments @mentioning their username

Reply:
＠megbird those are all great changes. I would just wait to see what direction  @brentjo and @benbalter want to take this.

Reply:
👋 @jidanni We have a process to ensure that pull requests are reviewed as quickly as we can, directly at-mentioning individuals will not get your pull request reviewed any more quickly and can actually slow the response due to people being distracted by notification noise.

`@brentjo` and `@benbalter` are not maintainers in this repository. You can add the changes @megbird suggested, as a maintainer of this repo, if you'd like to see an update to the docs.

Reply:
@janiceilene , I would love to add them. But whatever I press seems to mess things up. Perhaps you folks can add them for me. Thanks.